### PR TITLE
Re-enable some tests

### PR DESCRIPTION
**Pull request by Flakebi**
Fix the check-lines and re-enable some previously disabled tests.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 653a5fdfd8322328e15cdcaaf36579618460ff75
<details><summary>CTS tests (Failed: 0/207358)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 36986/69126 (53.5%)</li>
<li>Failed: 0/69126 (0.0%)</li>
<li>Not Supported: 32140/69126 (46.5%)</li><li>Warnings: 0/69126 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 37206/69104 (53.8%)</li>
<li>Failed: 0/69104 (0.0%)</li>
<li>Not Supported: 31898/69104 (46.2%)</li><li>Warnings: 0/69104 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 9.0, Gfx10</summary><ul>

<li>Passed: 34224/69128 (49.5%)</li>
<li>Failed: 0/69128 (0.0%)</li>
<li>Not Supported: 34904/69128 (50.5%)</li><li>Warnings: 0/69128 (0.0%)</li>
</ul></details>


</ul></details>

File: llpc/test/shaderdb/core/OpPtrDiff_Workgroup_mem.spvasm [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 ; SHADERTEST: [[c:%[0-9]+]] = sub i64 ptrtoint (ptr addrspace(3) @lds.1 to i64), [[b]]
 ; SHADERTEST: [[d:%[0-9]+]] = sdiv exact i64 [[c]], ptrtoint (ptr getelementptr (i64, ptr null, i32 1) to i64)
 ; SHADERTEST: [[e:%[0-9]+]] = trunc i64 [[d]] to i32
 
-; SHADERTEST: [[e1:%[0-9a-zA-Z.]+]] = bitcast i64 [[d1]] to <2 x i32>
-; SHADERTEST: {{%[0-9]+}} = insertelement <2 x i32> [[e1]], i32 [[e]], i64 1
+; SHADERTEST: [[i:%[0-9a-zA-Z.]+]] = insertelement <2 x i32> undef, i32 [[e1]], i32 0
+; SHADERTEST: {{%[0-9]+}} = insertelement <2 x i32> [[i]], i32 [[e]], i32 1
+
+; SHADERTEST-LABEL: }
```
Comment (nhaehnle) [inline llpc/test/shaderdb/core/OpPtrDiff_Workgroup_mem.spvasm:21]: I don't think this SHADERTEST-LABEL makes much sense?

Comment (Flakebi) [reply llpc/test/shaderdb/core/OpPtrDiff_Workgroup_mem.spvasm:21]: It makes sure that all the SHADERTEST checks are inside the same function. Without this label, half of the check lines above could match a different function than the second half.

Comment (nhaehnle) [reply llpc/test/shaderdb/core/OpPtrDiff_Workgroup_mem.spvasm:21]: Oh I see -- that's because FileCheck greedily scans for the label first. Thanks.

**Review by nhaehnle — APPROVED** [review]

**Pull request merged**